pages/tutorials/github_merge_queue.md

```diff
@@ -16,6 +16,12 @@ The merge queue creates temporary branches with a special prefix to validate pul
 
 To enable a merge queue for a pipeline:
 
+1. From your Buildkite dashboard, select your pipeline.
+1. Select _Pipeline Settings_ > _GitHub_.
+1. In the _GitHub Settings_ section, select the _Skip builds with existing commits_ checkbox.
+
+If you have existing branch limiting rules:
+
 1. From your Buildkite dashboard, select your pipeline.
 1. Select _Pipeline Settings_ > _GitHub_.
 1. In the _Branch Limiting_ section, add a filter for the following pattern:
```

Comment:
It would be good to avoid having to repeat the navigation instructions. What do you think about keeping it as one list, but outlining when you need to make changes to the branch limiting rules? Thinking:

```suggestion
1. In the _Branch Limiting_ section, check whether there are any existing branch filter patterns.
   - If none, you don't need to do anything further.
   - If there are existing patterns, add a filter for the following pattern:
```